fixtures/thomann/fun-generation-led-pot-12x1w-qcl-rgb-ww-40.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Fun Generation LED Pot 12x1W QCL RGB WW 40°",
```

Comment:
Manufacturer should be [Fun Generation](https://open-fixture-library.org/fun-generation)